plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferUtils.java

```diff
@@ -173,6 +181,14 @@ public static void refreshDatabaseModel(DBRProgressMonitor monitor, DatabaseCons
         return actions.toArray(new DBEPersistAction[0]);
     }
 
+    private static void getTableFullName(@Nullable DBSObjectContainer schema, @NotNull DBPDataSource dataSource, @NotNull StringBuilder sql, @NotNull String tableName) {
+        if (schema instanceof DBSSchema || schema instanceof DBSCatalog) {
+            sql.append(DBUtils.getQuotedIdentifier(schema));
+            sql.append(dataSource.getSQLDialect().getCatalogSeparator());
+        }
+        sql.append(DBUtils.getQuotedIdentifier(dataSource, tableName));
```

Comment:
Can we use `DBUtils#getFullQualifiedName(DBPDataSource, DBPNamedObject...)` here?